values.yaml

```diff
@@ -0,0 +1,177 @@
+# Default values for hyperopt.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+common:
+
+mongo:
+  image:
+    repository: mongo
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  nameOverride: ""
+  fullnameOverride: ""
+
+  port: 27017
+
+  trials_db: model_db
+  trials_collection: jobs
+
+  service:
+    type: ClusterIP
+
+  resources: {}
+    # If you want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #  cpu: 100m
+    #  memory: 128Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+  env: {}
+# Uncomment following lines to add env vars
+#    ABC: DEF
+
+
+monitor:
+  image:
+    repository: hipages/hyperkops-monitor
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  nameOverride: ""
+  fullnameOverride: ""
+
+  resources: {}
+    # If you want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #  cpu: 100m
+    #  memory: 128Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+  env:
+    TIMEOUT_INTERVAL: 15
```

Comment:
remove the timeout interval

Reply:
These are the arguments (and defaults) required for the monitor:


mongo_db_address | MONGO_DB_ADDRESS | url to MongoDB | localhost
mongo_db_port | MONGO_DB_PORT | mongo db port | 27017
trials_db | TRIALS_DB | Name of the MongoDB in which the trials are stored | model_db
trials_collection | TRIALS_COLLECTION | Name of the Mongo Collection in which the trials are stored | jobs
update_interval | UPDATE_INTERVAL | Time between queries to the MongoDB to find failed jobs | 100
namespace | NAMESPACE | Namespace in which the Pods to be monitored are being deployed |  
label-selector | LABEL_SELECTOR | Labels which identify relevant Hyperkops worker Pods

Reply:
https://github.com/hipagesgroup/hyperkops/blob/DATA-2526/examples/kube-deploy-hyperkops-infrastructure.yaml
this might help